### PR TITLE
Add second model with other train parameter

### LINKED ISSUE #63: Add second served model in order to create more realistic environment

### PR DESCRIPTION
**Pull request by avan1235**
Closes #63

### CODE REVIEW

File: dev-environment/mlflow/run.sh [hunk 1]
```diff
@@ -2,18 +2,33 @@
...
+  if [ $? -eq 1 ]; then
+    echo "Creating new mlflow experiment with id $1"
+    mlflow experiments create --experiment-name "Test experiment $1" --artifact-location "s3://mlflow/$1"
+  fi
+}
+
+create_experiment_if_not_exists 1 &&
+create_experiment_if_not_exists 2
+
+create_mlflow_run 0.42 0.5 $MODEL_1_PORT 1 &
```
Comment (kantoniak) [inline dev-environment/mlflow/run.sh:33]: Background task - intentional?

File: prinz/src/test/scala/pl/touk/nussknacker/prinz/UnitIntegrationTest.scala [hunk 2]
```diff
@@ -23,7 +24,9 @@ object UnitIntegrationTest {
 
   private val DOCKER_COMPOSE_FILE = new File("../dev-environment/docker-compose.yaml")
 
-  private val MODEL_SERVED_READY_REGEX = s".*Listening at.*1234.*"
+  private val MODEL_1_SERVED_READY_REGEX = s".*Listening at.*1234.*"
+
+  private val MODEL_2_SERVED_READY_REGEX = s".*Listening at.*4321.*"
```
Comment (kantoniak) [inline prinz/src/test/scala/pl/touk/nussknacker/prinz/UnitIntegrationTest.scala:29]: Can we take port number from somewhere?

**Review by kantoniak — APPROVED** [review]

File: dev-environment/mlflow/run.sh (hunk 1 above)
Comment (avan1235) [reply dev-environment/mlflow/run.sh:33]: yes @kantoniak, calling this function makes it listening as a server with model so we need to start one model-server in background and the second one in foreground to keep docker working

File: prinz/src/test/scala/pl/touk/nussknacker/prinz/UnitIntegrationTest.scala (hunk 2 above)
Comment (avan1235) [reply prinz/src/test/scala/pl/touk/nussknacker/prinz/UnitIntegrationTest.scala:29]: we can assume it is defined in env variables and the n the tests in scala as well as docker env would take the same value, to be done in future if needed

**Pull request merged**